### ISSUE TITLE
Compound searches using SMILES etc

### ISSUE THREAD

**Issue by Emma-Manners** [chembl/GLaDOS#1256]
**Consider adding:**

SMILES list search functionality on the ChEMBL interface
List search for names, InChI key, ChEMBL_ID

**Comment by nclopezo** [comment]
Yes, I have been thinking on adding that functionality for some time. But I always get busy with other things! Sorry

**Comment by Emma-Manners** [comment]
No worries, just had a couple of questions on this so thought I'd add it!
